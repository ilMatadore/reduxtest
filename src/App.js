import logo from './logo.svg';
import './App.css';
import Header from './components/header/header.component';
import React from 'react';

class App extends React.Component {


  render() {   
    return (
      <div className="App">
        <header className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <Header />
        </header>
      </div>
    );
    }
}

export default (App);
