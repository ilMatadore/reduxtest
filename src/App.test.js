import { shallow} from 'enzyme';
import App from './App';

it('render App', () => {
  expect(shallow(<App />)).toMatchSnapshot();
})
