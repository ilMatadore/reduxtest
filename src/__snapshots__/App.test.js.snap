// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <Connect(Header) />
  </header>
</div>
`;
